policy/modules/system/authlogin.te

```diff
@@ -128,6 +128,8 @@ dev_read_urand(chkpwd_t)
 dev_search_sysfs(chkpwd_t)
 
 files_read_etc_files(chkpwd_t)
+# to read /etc/machine-id
+files_read_etc_runtime_files(chkpwd_t)
```

Comment:
This is presumably related to pam_systemd; if so, we should add access in the related interface or call it.